Packs/CheckPhish/Integrations/CheckPhish/CheckPhish.py

```diff
@@ -196,37 +194,67 @@ def get_result_checkphish(jobID):
         return_error('Error getting job status')
 
 
-def checkphish_check_urls():
+def checkphish_check_urls(**kwargs):
     urls = argToList(demisto.args().get('url'))
     job_ids = []
 
     for url in urls:
-        submit = submit_to_checkphish(url)
+        submit = submit_to_checkphish(url, kwargs.get('api_key'), kwargs.get('base_url'), kwargs.get('use_ssl'))
         if submit:
             job_ids.append(submit)
 
     while len(job_ids):
         for job_id in job_ids[:]:
-            if is_job_ready_checkphish(job_id):
-                get_result_checkphish(job_id)
+            if is_job_ready_checkphish(job_id, kwargs.get('api_key'), kwargs.get('base_url'), kwargs.get('use_ssl')):
+                get_result_checkphish(job_id, kwargs.get('api_key'), kwargs.get('base_url'), kwargs.get('use_ssl'),
+                                      kwargs.get('reliability'))
                 job_ids.remove(job_id)
 
 
 ''' COMMANDS MANAGER / SWITCH PANEL '''
 
 LOG('Command being called is %s' % (demisto.command()))
 handle_proxy()
-unite_dispositions()
 
-try:
-    if demisto.command() == 'test-module':
-        demisto.results(test_module())
 
-    elif demisto.command() == 'CheckPhish-check-urls':
-        checkphish_check_urls()
+def main():
 
-# Log exceptions
-except Exception as e:
-    LOG(str(e))
-    LOG.print_log()
-    raise
+    params = demisto.params()
+
+    good_disp = argToList(params.get('good_disp'))
+    susp_disp = argToList(params.get('susp_disp'))
+    bad_disp = argToList(params.get('bad_disp'))
+
+    unite_dispositions(good_disp, susp_disp, bad_disp)
+
+    reliability = params.get('integrationReliability')
+    reliability = reliability if reliability else DBotScoreReliability.B
+
+    if DBotScoreReliability.is_valid_type(reliability):
+        reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
+    else:
+        raise Exception("Please provide a valid value for the Source Reliability parameter.")
+
+    args = {
+        'base_url': params['url'],
+        'api_key': params.get('token'),
+        'use_ssl': not params.get('insecure', False),
+        'reliability': reliability
+    }
```

Comment:
AS we talked about in previous PRs, please separate params from args, and pass both as dicts to the commands.